### ISSUE TITLE
Include mappyfile in conda-forge?

### ISSUE THREAD

**Issue by tchaddad** [geographika/mappyfile#165]
Not sure if this is an appropriate topic for a GitHub issue, but I am curious if it would be possible to make mappyfile available via conda-forge (https://conda-forge.org) so that it can be easily utilized by Anaconda users. Thanks for considering!

**Comment by geographika** [comment]
@tchaddad - fine to have this an issue. I've not used conda personally, so not sure how it works beyond a quick glance at https://conda-forge.org/docs/maintainer/adding_pkgs.html#

If anyone has any experience of adding packages to conda I'm happy to add any files to this repository if they are required.

**Comment by tchaddad** [comment]
@geographika apologies for slow reply, I finally researched this and found a [tutorial for the required steps](https://blog.gishub.org/how-to-publish-a-python-package-on-conda-forge). Based on the process outlined there, I've created a [recipe for adding mappyfile to conda-forge](https://github.com/conda-forge/staged-recipes/compare/main...tchaddad:staged-recipes:main). 

I set the python version to >=3.8 based on the requirements listed on the mappyfile GitHub README. The other requirements listed were populated based on the mappyfile requirements pulled from PyPI

At the bottom of the recipe there is a space for 'recipe-maintainers'. You can have multiple, and the pull request checklist says "GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there", so if you respond here agreeing to be listed, I guess we can point to that comment to check that box, and add you to the recipe.

Please take a look and see if it looks correct / complete to you, and if so, someone can submit the recipe as a pull request to the conda-forge/staged-recipes repo.

**Comment by geographika** [comment]
@tchaddad - thanks for this, and great timing! I'm currently trying to get an official 1.0 release together, and applying for mappyfile to be an OSGeo Community project. I'm happy to be added to the maintainer section. 

The config in the pull request looks fine as far as I can tell (and looks similar to the [pygeoapi](https://github.com/conda-forge/staged-recipes/pull/10592/files#diff-8d462abfc2ee995ff1e8eee7c1386f94eae9029d565a466ee62bc657275e1a79) metadata which was recently added). It also seems simple to keep updated. 

Would you be able to submit the pull request to conda-forge?

**Comment by tchaddad** [comment]
Great, I have added you to the maintainers, and will submit a pull request. From what I understand, there will next be some auto tests, and then a final review by conda-forge maintainers. I'll update here on any issues or next steps...

**Comment by tchaddad** [comment]
Errors on this first attempt, and need a bit of time to dive into the cause. You can [browse the results here if curious](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=751919&view=results). Will circle back in a bit with a few changes...

**Comment by tchaddad** [comment]
OK we are [passing all checks now](https://github.com/conda-forge/staged-recipes/pull/23490), so I've requested the review step...

**Comment by tchaddad** [comment]
The last round of review resolved all the outstanding issues and so the mappyfile recipe was merged into [the conda-forge](https://conda-forge.org/feedstock-outputs/) earlier today and [is now visible there](https://anaconda.org/conda-forge/mappyfile/files). I tested locally and confirmed that installing mappyfile via Conda does indeed now work, so that's an improvement for folks who like to work with those tools. Looks like maintaining the connection [should not bee too complicated](https://github.com/conda-forge/staged-recipes/pull/23490#issuecomment-1682418610). Thanks for your help!
